symphony-bdk-core/src/test/java/com/symphony/bdk/core/service/message/util/MessageUtilTest.java

```diff
@@ -0,0 +1,35 @@
+package com.symphony.bdk.core.service.message.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class MessageUtilTest {
```

Comment:
parameterized tests could be an option here too
(but I'm ok with the ones below, it is easy to understand)

Reply:
If it's ok I would prefer to keep it like this. Since there are not so much tests and also they are really small.